clustermesh-apiserver/main.go

```diff
@@ -432,19 +427,20 @@ func updateEndpoint(obj interface{}) {
 
 				keyPath := path.Join(ipcache.IPIdentitiesPath, ipcache.DefaultAddressSpace, ip)
 				entry := identity.IPIdentityPair{
-					IP:           net.ParseIP(ip),
+					IP: net.ParseIP(ip),
+					// FIXME: @jrajahalme this should have the VM name?
```

Comment:
VMs should not stand out from regular endpoints here. VMs name should already be in the K8sPodName field?